x-pack/plugins/aiops/server/routes/explain_log_rate_spikes.ts

```diff
@@ -172,11 +177,22 @@ export const defineExplainLogRateSpikesRoute = (
         for (const fieldCandidatesChunk of fieldCandidatesChunks) {
           let pValues: Awaited<ReturnType<typeof fetchChangePointPValues>>;
           try {
-            pValues = await fetchChangePointPValues(client, request.body, fieldCandidatesChunk);
+            pValues = await fetchChangePointPValues(
+              client,
+              request.body,
+              fieldCandidatesChunk,
+              logger,
+              pushError
+            );
           } catch (e) {
-            push(addErrorAction(e.toString()));
-            end();
-            return;
+            logger.error(
+              `Failed to fetch p-values for ${JSON.stringify(
+                fieldCandidatesChunk
+              )}, got: \n${e.toString()}`
+            );
+            pushError(`Failed to fetch p-values for ${JSON.stringify(fieldCandidatesChunk)}.`);
```

Comment:
Should this message be internationalized?

Reply:
We have an item in the meta issue for 8.6 to pick this up, it affect all messages in this file, not just the updated ones, will do in a follow up: https://github.com/elastic/kibana/issues/141214